iotdb-core/datanode/src/main/java/org/apache/iotdb/db/subscription/event/response/SubscriptionEventTsFileResponse.java

```diff
@@ -234,7 +234,7 @@ private void waitForResourceEnough4Slicing(final long timeoutMs) throws Interrup
       if (waitTimeSeconds * 1000 > timeoutMs) {
         // should contain 'TimeoutException' in exception message
         // see org.apache.iotdb.rpc.subscription.exception.SubscriptionTimeoutException.KEYWORD
-        throw new InterruptedException(
+        throw new SubscriptionException(
```

Comment:
Remove all InterruptedException flags from the call stack, like https://github.com/apache/iotdb/blob/master/iotdb-core/datanode/src/main/java/org/apache/iotdb/db/subscription/event/response/SubscriptionEventTsFileResponse.java#L115, https://github.com/apache/iotdb/blob/master/iotdb-core/datanode/src/main/java/org/apache/iotdb/db/subscription/event/response/SubscriptionEventTsFileResponse.java#L120, https://github.com/apache/iotdb/blob/master/iotdb-core/datanode/src/main/java/org/apache/iotdb/db/subscription/event/response/SubscriptionEventTsFileResponse.java#L120